testinfra/modules/package.py

```diff
@@ -165,7 +165,13 @@ def version(self):
 class RpmPackage(Package):
     @property
     def is_installed(self):
-        return self.run_test("rpm -q %s", self.name).rc == 0
+        result = self.run("rpm -q --quiet %s 2>&1", self.name)
+        if result.succeeded:
+            return True
+        elif result.failed and result.stdout == '':
+            return False
+        else:
+            raise RuntimeError(f"Could not check if RPM package '{self.name}' is installed. {result.stdout}")
```

Comment:
FYI: If return code is `1` and there is `stdout`, that means there was an unexpected error, so we throw a `RuntimeError` here. (as we use `--quiet` there should not be any output when failing).